### PR TITLE
add new rubocops and concomittant tweaks to code

### PR DESCRIPTION
**Pull request by ndushay**
## Why was this change made?

:shrug:

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -1,5 +1,22 @@
+require:
+  - rubocop-rspec
+
 inherit_from: .rubocop_todo.yml
 
+AllCops:
+  TargetRubyVersion: 3.1
+  SuggestExtensions: false
+  DisplayCopNames: true
+  Include:
```
Comment (justinlittman) [inline .rubocop.yml:10]: Can you remove includes/excludes for files that don't exist in this repo?

**Review by justinlittman — CHANGES_REQUESTED** [review]

File: .rubocop.yml (hunk 1 above)
Comment (ndushay) [reply .rubocop.yml:10]: I guess I was thinking that the boilerplate is harmless enough, and we do have rake files ...

Comment (justinlittman) [reply .rubocop.yml:10]: For me, boilerplate is fine in files that we never look at, but counts as cruft when in files that we regularly touch/think about.

Comment (ndushay) [reply .rubocop.yml:10]: removed!

**Review by justinlittman — APPROVED** [review]

**Comment by justinlittman** [comment]
Thanks for considering!

**Pull request merged**